intermediate_source/process_group_cpp_extension_tutorial.rst

```diff
@@ -0,0 +1,252 @@
+Customize Process Group Backends Using Cpp Extensions
+=====================================================
+
+**Author**: `Feng Tian <https://github.com/ftian1>`__, `Shen Li <https://mrshenli.github.io/>`_-
```

Comment:
Hey @ftian1, do you mind if I add you as the first author of this tutorial? BTW, thanks a lot for contributing this feature!

Reply:
no problem:) It's my pleasure to be in here